cetmix_tower_server/models/cx_tower_file_template.py

```diff
@@ -53,27 +57,35 @@ def _compute_file_count(self):
         required=True,
         default="tower",
     )
-
     variable_ids = fields.Many2many(
         comodel_name="cx.tower.variable",
         relation="cx_tower_file_template_variable_rel",
```

Comment:
We can drop "relation", "column1" and "column2" too. They will be created automatically.